components/chips.css

```diff
@@ -0,0 +1,69 @@
+.eg-chips {
+    align-items: center;
+    display: flex;
+    flex-wrap: wrap;
+    gap: 10px;
+    margin-top: var(--size-12);
+}
+
+.chip {
+    align-items: center;
+    background-color: var(--primary-100);
+    border-radius: var(--size-28);
+    box-shadow: 0 0 2px var(--primary-300);
+    display: flex;
+    gap: 5px;
+    justify-content: space-between;
+    padding: 8px 8px;
```

Comment:
I'll suggest to use only one value if you're giving same values. Also you can use a variable here.

Reply:
Yeah didn't realize that. Maybe amidst tweaking for different values I might have settled for the same unknowingly. 
Done✅